pylabrobot/tilting/hamilton_backend.py

```diff
@@ -15,8 +15,9 @@ def __init__(self, com_port: str, write_timeout: float = 10, timeout: float = 10
     self.serial: Optional[serial.Serial] = None
     self.timeout = timeout
     self.write_timeout = write_timeout
+    self.ser: Optional[serial.Serial] = None
 
-  async def setup(self):
+  async def setup(self, initial_offset: int=0):
```

Comment:
I added this because our box kept initializing to an angle of ~5 degrees as zero. I initialize this offset with an angle of -20 degrees, and then it homes correctly to 0.

Reply:
ah, that's the thing I vaguely mentioned to JJ. thanks for finding this :)